validator/src/main.rs

```diff
@@ -1443,9 +1443,9 @@ pub fn main() {
                 .help("Enable an accounts index, indexed by the selected account field"),
         )
         .arg(
-            Arg::with_name("accounts_db_caching_enabled")
-                .long("accounts-db-caching-enabled")
-                .help("Enable accounts caching"),
+            Arg::with_name("accounts_db_caching_disabled")
+                .long("accounts-db-caching-disabled")
```

Comment:
A _no_ prefix is more inline with the rest of of the solana-validator flags that disable features:
```suggestion
                .long("no-accounts-db-caching")
```